### DISCUSSION TITLE
Edit user profile

### DISCUSSION THREAD

**Discussion by florincretzan** [codeigniter4/shield#571] [Q&A]
Hello guys,

What is the best way to update user profile. username, email, password, etc ?

I've managed to do it but I think this functionality shoud be enabled out of the box !

Thank you !

**Comment by kenjis** [comment]
I recommend you check https://github.com/lonnieezell/Bonfire2

Shield is Auth library. It is not user management system.

**Comment by datamweb** [comment]
Hi, see [editing user ](https://codeigniter4.github.io/shield/quickstart/#editing-a-user)
